ntoskrnl/se/debug.c

```diff
@@ -107,24 +107,23 @@ SepDumpAces(
              * Just fail gracefully and stop further
              * debugging of ACEs.
              */
-            DbgPrint("SepDumpAces(): Failed to find the next ACE, stop dumping info...\n");
+            DPRINT("SepDumpAces(): Failed to find the next ACE, stop dumping info...\n");
             return;
         }
 
-        DbgPrint("================== %lu# ACE DUMP INFO ==================\n", AceIndex);
-        DbgPrint("Ace -> 0x%p\n", Ace);
-        DbgPrint("Ace->Header -> 0x%p\n", Ace->Header);
```

Comment:
@GeoB99 I removed this line, because you were printing an object as a pointer.
You probably wanted to print the address of Header, but that's the same as `Ace` since it's the first item.